### PR TITLE
Convnet aliases

### PR DESCRIPTION
**Pull request by farizrahman4u**
Not very important, but kinda handy.

### CODE REVIEW

File: keras/layers/convolutional.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
-class AtrousConv2D(Convolution2D):
+class AtrousConvolution2D(Convolution2D):
```
Comment (fchollet) [inline keras/layers/convolutional.py]: In that case this name should also be modified in all calls to it with the class definition (super).

Comment (farizrahman4u) [inline keras/layers/convolutional.py]: Ok

File: keras/layers/convolutional.py [hunk 2]
```diff
@@ -390,7 +390,7 @@ class AtrousConv2D(Convolution2D):
     ```python
         # apply a 3x3 convolution with atrous rate 2x2 and 64 output filters on a 256x256 image:
         model = Sequential()
-        model.add(AtrousConv2D(64, 3, 3, atrous_rate=(2,2), border_mode='valid', input_shape=(3, 256, 256)))
+        model.add(AtrousConvolution2D(64, 3, 3, atrous_rate=(2,2), border_mode='valid', input_shape=(3, 256, 256)))
```
Comment (fchollet) [inline keras/layers/convolutional.py]: Not necessary here

File: keras/layers/convolutional.py [hunk 3]
```diff
@@ -460,7 +460,7 @@ def __init__(self, nb_filter, nb_row, nb_col,
 
         self.atrous_rate = tuple(atrous_rate)
 
-        super(AtrousConv2D, self).__init__(nb_filter, nb_row, nb_col,
+        super(AtrousConvolution2D, self).__init__(nb_filter, nb_row, nb_col,
```
Comment (fchollet) [inline keras/layers/convolutional.py]: PEP8 (indentation)

**Comment by fchollet** [comment]
LGTM

**Pull request merged**